src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -163,6 +167,24 @@ public function text($message)
     /**
      * {@inheritdoc}
      */
+    public function comment($message)
+    {
+        $this->autoPrependText();
+
+        if (!is_array($message)) {
```

Comment:
I think it could be simplify to something like:
```php
$this->autoPrependText();
$messages = is_array($messages) ? array_values($messages) : array($messages);
foreach ($messages as $message) {
     $this->writeln(sprintf(' // %s', $message));
}
```

Unless you really need the recursion to do something like:
```php
$sfStyle->comment([
    ['lorem', ['ipsum', 'dolor']],
    'Sit',
    'amet',
])
```